internal/rbd/rbd_util.go

```diff
@@ -206,7 +206,18 @@ var (
 			needRbdNbd: false,
 		},
 		librbd.FeatureNameExclusiveLock: {
-			needRbdNbd: true,
+			needRbdNbd: false,
+		},
+		librbd.FeatureNameObjectMap: {
+			needRbdNbd: false,
+			dependsOn:  []string{librbd.FeatureNameExclusiveLock},
+		},
+		librbd.FeatureNameFastDiff: {
+			needRbdNbd: false,
+			dependsOn:  []string{librbd.FeatureNameObjectMap},
+		},
+		librbd.FeatureNameDeepFlatten: {
```

Comment:
I don't see e2e for deepflatten feature can you please include this in e2e also?

Reply:
@k0ste any specific reason for adding deep-flatten image feature? currently deep flatten is handled internally by cephcsi to make clones and snapshots independent of each other

Reply:
Deep-flatten may be needed by external administration features, out of Kubernetes scope

Reply:
as I mentioned earlier deep flattening is already handled by cephcsi, I think we can have a separate PR with the use-case that it will solve.